mailbox/tika/src/main/java/org/apache/james/mailbox/tika/TikaHttpClientImpl.java

```diff
@@ -57,6 +60,7 @@ public InputStream recursiveMetaDataAsJson(InputStream inputStream, String conte
                     .returnContent()
                     .asStream();
         } catch (IOException e) {
+            LOGGER.error("Failing to call Tika", e);
```

Comment:
why do you need to log here if you throw? It should be logged by the caller, no?

Reply:
ok fixed later